node/network/statement-distribution/src/lib.rs

```diff
@@ -532,6 +539,15 @@ async fn circulate_statement_and_dependents(
 		None => return,
 	};
 
+	let _span = {
+		let mut span = active_head.span.child("circulate-statement");
+		span.add_string_tag(
```

Comment:
I'm no fan of all of these calls. We will hash the payload and generate a hash, even if there is no Jaeger instance running, meaning we waste a lot of cpu power.

Reply:
https://github.com/paritytech/polkadot/issues/2150